### PR TITLE
jsonnet: Add config to support rhelemeter

### PR DESCRIPTION
**Pull request by philipgough**
https://issues.redhat.com/browse/RHOBS-789

### CODE REVIEW

File: jsonnet/telemeter/server/rhelemeter-kubernetes.libsonnet [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+      resourceLimits: {},
+      resourceRequests: {},
+    },
+
+    versions+:: {
+      rhelemeterServer: 'v4.0',
+    },
+
+    imageRepos+:: {
+      rhelemeterServer: 'quay.io/openshift/origin-telemeter',
```
Comment (philipgough) [inline jsonnet/telemeter/server/rhelemeter-kubernetes.libsonnet:75]: rhelemeter shares the image and runs a different binary so this remains the same as telemeter

**Review by saswatamcode — APPROVED** [review]
/lgtm

File: jsonnet/telemeter/server/rhelemeter-kubernetes.libsonnet [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+        $._config.rhelemeterServer.elideLabels
+      );
+
+
+      local rhelemeterServer =
+        container.new('rhelemeter-server', $._config.imageRepos.rhelemeterServer + ':' + $._config.versions.rhelemeterServer) +
+        container.withCommand([
+          '/usr/bin/rhelemeter-server',
+          '--listen=0.0.0.0:8443',
+          '--listen-internal=0.0.0.0:8081',
```
Comment (thibaultmg) [inline jsonnet/telemeter/server/rhelemeter-kubernetes.libsonnet:115]: We could use the ports variables here

**Review by thibaultmg — APPROVED** [review]
lgtm

**Review by thibaultmg — APPROVED** [review]
lgtm

**Pull request merged**